Assets/MixedRealityToolkit.Tests/EditModeTests/Core/Utilities/PackageManifestUpdaterTests.cs

```diff
@@ -0,0 +1,164 @@
+        /// </summary>
+        [Test]
+        public void OutdatedMSBuild()
+        {
+            Debug.Log("Upgrade to new verion / prerelease");
+            string minVersion = "1.5.22-20200919.28";
+            string currentVersion = "0.7.15";
+
+            bool isAppropriate = PackageManifestUpdater.IsAppropriateMBuildVersion(minVersion, currentVersion);
+            Assert.IsFalse(isAppropriate);
+
+            Debug.Log("Upgrade to new prerelease");
+            minVersion = "9.4.2-20200622.4";
+            currentVersion = "9.4.2-20200530.9";
+            isAppropriate = true;
+
+            isAppropriate = PackageManifestUpdater.IsAppropriateMBuildVersion(minVersion, currentVersion);
+            Assert.IsFalse(isAppropriate);
+
+            Debug.Log("Upgrade to new version");
+            minVersion = "1.0.0";
+            currentVersion = "0.7.15";
+            isAppropriate = true;
+
+            isAppropriate = PackageManifestUpdater.IsAppropriateMBuildVersion(minVersion, currentVersion);
+            Assert.IsFalse(isAppropriate);
+
+            Debug.Log("Upgrade prerelease to final");
+            minVersion = "0.9.19";
+            currentVersion = "0.9.19-20200101.55";
+            isAppropriate = true;
+
+            isAppropriate = PackageManifestUpdater.IsAppropriateMBuildVersion(minVersion, currentVersion);
+            Assert.IsFalse(isAppropriate);
+        }
+
+        /// <summary>
+        /// Verifies that an exactly matching MSBuild for Unity version returns the correct result.
+        /// </summary>
+        [Test]
+        public void MatchingMSBuild()
+        {
+            Debug.Log("Compare version");
+            string minVersion = "28.32.44";
+            string currentVersion = "28.32.44";
+
+            bool isAppropriate = PackageManifestUpdater.IsAppropriateMBuildVersion(minVersion, currentVersion);
+            Assert.IsTrue(isAppropriate);
+
+            Debug.Log("Compare version / prerelease");
+            minVersion = "1.5.22-20200919.28";
+            currentVersion = "1.5.22-20200919.28";
+            isAppropriate = false;
+
+            isAppropriate = PackageManifestUpdater.IsAppropriateMBuildVersion(minVersion, currentVersion);
+            Assert.IsTrue(isAppropriate);
+        }
+
+        /// <summary>
+        /// Verifies that a more recent MSBuild for Unity version returns the correct result.
+        /// </summary>
+        [Test]
+        public void NewerMSBuild()
+        {
+            Debug.Log("Newer version");
+            string minVersion = "28.32.44";
+            string currentVersion = "28.32.45";
+
+            bool isAppropriate = PackageManifestUpdater.IsAppropriateMBuildVersion(minVersion, currentVersion);
+            Assert.IsTrue(isAppropriate);
+
+            Debug.Log("Newer prerelease");
+            minVersion = "1.5.22-20200919.28";
+            currentVersion = "1.5.22-20200919.29";
+            isAppropriate = false;
```

Comment:
The re-setting of isAppropriate here and below isn't necessary (because they are then immediately set to the output of IsAppropriateMBuildVersion)

Reply:
wanted to make certain the method was changing the value. i can remove :smile:

Reply:
done

Reply:
It's definitely changing the value right? It's not getting it as an input